package.json

```diff
@@ -67,7 +67,7 @@
     "eslint-plugin-react": "~7.0.0",
     "eslint-plugin-react-native": "^2.2.1",
     "eslint-plugin-spellcheck": "~0.0.6",
-    "flow-bin": "~0.40.0",
+    "flow-bin": "^0.40.0",
```

Comment:
Looks like it's still matching any minor version greater than 0.40? Let's leave the `~` for now and change it in another diff